airbyte-workers/Dockerfile

```diff
@@ -1,11 +1,12 @@
-ARG JDK_VERSION=17.0.4
+ARG JDK_VERSION=17-al2023
 ARG JDK_IMAGE=amazoncorretto:${JDK_VERSION}
 FROM ${JDK_IMAGE} AS worker
 
 ARG DOCKER_BUILD_ARCH=amd64
 
-RUN amazon-linux-extras install -y docker
-RUN yum install -y jq tar && yum update --security -y && yum clean all
+RUN dnf install -y docker
+RUN dnf install -y jq tar && dnf update --security -y && dnf clean all
+RUN dnf install -y containerd --releasever=2023.0.20230503
```

Comment:
You should be able to simplify these 3 lines as:

```RUN dnf install -y docker containerd jq --releasever=latest```

```dnf update --security -y``` is not longer required, and doesn't actually update anything, because Amazon no longer any includes any security updates in a given Amazon Linux 2023 package repo. All repos are now effectively immutable. Setting `--releasever=latest` is the closest thing available to say "pull whatever is the latest update for the given package for AL2023".